lib/SIL/Verifier/LinearLifetimeChecker.cpp

```diff
@@ -707,7 +706,13 @@ bool LinearLifetimeChecker::usesNotContainedWithinLifetime(
   assert(numFoundUses == uniqueUsers.size());
 #endif
 
-  // Return true if we /did/ found an error and when emitting that error, we
+  // If we found any error except for uses outside of our lifetime, bail.
+  if (error.getFoundLeak() || error.getFoundOverConsume() ||
+      error.getFoundUseAfterFree())
```

Comment:
Eventually I may want to have some sort of enum here. Makes me nervous doing these sorts of exhaustive checks. But this code doesn't change too much so I think it is ok for now.

Reply:
FWIW, I find it easier to think about APIs that don't involve double-negative. i.e. is should be '!usesContainded', not 'usesNotContained'

Reply:
Ok. I am fine with changing that.

Reply:
I am going to invert that in a follow on commit.